react-view/src/pages/PostHistoryTablePage/PostsTable/PaginationButtons.tsx

```
@@ -0,0 +1,82 @@
+        {displayedPages.map((page, index) =>
+          page ? (
+            <Button
+              key={index}
+              variant={page === currentPage ? "solid" : "ghost"}
+              aria-label={"page number " + page}
+              onClick={() => handlePageChange(Number(page))}
+            >
+              {page}
+            </Button>
+          ) : (
+            <Text>...</Text>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)